requirements.dev.conda.yml

```diff
@@ -1,4 +1,4 @@
-name: tiatoolbox
+name: tiatoolbox_dev
```

Comment:
I slightly prefer - to _  just because it's easier to type when activating, but not a big deal

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/198/commits/7b4dc3bdecf065b587fbd97264cd7c824861cf2d